packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -492,4 +447,104 @@ pc.close = (peerConnection) => {
     });
 };
 
+
+pc.setPeerConnectionEvents = (meeting) => {
+  // In case ICE fail
+  const {peerConnection} = meeting.mediaProperties;
+
+  const connectionFailed = () => {
+    if (meeting.reconnectionManager.iceState.resolve) {
+      // DISCONNECTED state triggers first then it goes to FAILED STATE
+      // sometimes the failed state can happen before 10 seconds (Which is the timer for the reconnect for ice disconnect)
+      meeting.reconnectionManager.iceState.resolve();
+    }
+
+    meeting.reconnect({networkDisconnect: true});
+    Metrics.postEvent({
+      event: eventType.ICE_END,
+      meeting,
+      data: {
+        canProceed: false,
+        errors: [
+          Metrics.generateErrorPayload(
+            2004, false, error.name.MEDIA_ENGINE
+          )]
+      }
+    });
+  };
+
+  peerConnection.oniceconnectionstatechange = () => {
+    LoggerProxy.logger.info('Meeting:util#setPeerConnectionEvents --> ICE STATE CHANGE.');
```

Comment:
All of these moved logs need to be: 
```suggestion
    LoggerProxy.logger.info('PeerConnectionManager:index#setPeerConnectionEvents --> ICE STATE CHANGE.');
```